src/flick/suggestion/models.py

```diff
@@ -9,16 +9,18 @@ class PublicSuggestion(models.Model):
     author = models.ForeignKey(Profile, related_name="suggestions_posted", on_delete=models.CASCADE)
     message = models.CharField(max_length=140, blank=True)
     show = models.ForeignKey(Show, related_name="suggestion_post", blank=False, on_delete=models.CASCADE)
-    created_at = models.DateTimeField(auto_now=True)
+    created_at = models.DateTimeField(auto_now_add=True)
+    updated_at = models.DateTimeField(auto_now=True)
```

Comment:
what's the difference between `auto_now_add` and `auto_now `

Reply:
and `updated_at` changes when the suggestion is viewed? how is it being used

Reply:
yahh so auto_now_add is when you add the object the first time, and auto_now is just anytime you update the object!!

Reply:
so the idea is that `updated_at` will automatically be updated as the last time you updated this `suggestion` object, and this helps me decide whether or not i should remove some suggestions from the `num_notifs` count (bc if you've already seen the suggestion at `suggest_time_viewed` then we need to remove the number (`num_notifs`) in the red bubble)!!